sql-plugin/src/main/scala/org/apache/spark/sql/rapids/AggregateFunctions.scala

```diff
@@ -286,7 +286,7 @@ abstract class GpuDeclarativeAggregate extends GpuAggregateFunction with GpuUnev
 
   // mostly likely a pass through (count => sum we merged above).
   // average has a more interesting expression to compute the division of sum/count
-  val evaluateExpression: Expression
+  val evaluateExpression: Seq[Expression]
```

Comment:
I think we need probably should rename this to make it plural. `evaluateExpressions`